Demo/src/main/java/com/paypal/android/PayPalConfigConstants.kt

```diff
@@ -0,0 +1,5 @@
+package com.paypal.android
+
+class PayPalConfigConstants {
+    val returnUrl = "${BuildConfig.APPLICATION_ID}://paypalpay"
```

Comment:
do we still need it, in iOS it works without returnUrl

Reply:
yes, android integration still needs a returnUrl